backend/src/main/java/com/google/rolecall/models/Section.java

```diff
@@ -181,8 +190,17 @@ public static Builder newBuilder() {
     private String name;
     private String notes;
     private Integer length;
+    private Integer siblingId;
     private Type type;
 
+    public Builder setId(Integer id) {
```

Comment:
ID is a GenerationType.AUTO and I do not think we should allow manual value here.

Reply:
What I am trying to do is to update the siblingId or name fields of an existing section record (the sibling record). Because of the builder pattern, I can't update the record directly that is currently in memory. So I have to create a new one. If I can't set the Id of that record, I will create a new record in a new location, which would be wrong because what I want to do is to update an existing record. If I can't set the ID of the record I just created, which pattern or code should I use?

Reply:
I looked more into the code. I would have to write a lot of code to get around the fact that I can't set the Id directly. What if rename setId(Integer id) to setIdUnsafe(Integer id)?

Reply:
setIdUnsafe does not match the setter pattern. If we are going to keep it here, it should be Section#setId. Can we please add an inline comment about why we need it?